qa-include/db/install.php

```diff
@@ -107,7 +112,7 @@ function qa_db_table_definitions()
 			'avatarheight' => 'SMALLINT UNSIGNED', // pixel height of stored avatar
 			'passsalt' => 'BINARY(16)', // salt used to calculate passcheck - null if no password set for direct login
 			'passcheck' => 'BINARY(20)', // checksum from password and passsalt - null if no password set for direct login
-			'passhash' => 'VARCHAR(255) CHARACTER SET utf8 COLLATE utf8_bin DEFAULT NULL', // password_hash
+			'passhash' => 'VARCHAR(255) CHARACTER SET utf8 COLLATE '.$collation.' DEFAULT NULL', // password_hash
```

Comment:
You also need to change the charset if you change the collation, apparently...

Otherwise you may get an error like:
> COLLATION 'utf8_bin' is not valid for CHARACTER SET 'utf8mb4'

(At least I got it in MariaDB)

```suggestion
			'passhash' => 'VARCHAR(255) CHARACTER SET utf8mb4 COLLATE '.$collation.' DEFAULT NULL', // password_hash
```

(Of course may be adjusted to work with the defined `QA_USE_UTF8MB4` option.)